services/wfm/src/main/java/org/openkilda/wfm/topology/flowhs/service/FlowCreateService.java

```diff
@@ -73,6 +73,9 @@ public void handleRequest(String key, CommandContext commandContext, FlowRequest
         if (requestedFlow.getFlowEncapsulationType() == null) {
             requestedFlow.setFlowEncapsulationType(kildaConfigurationRepository.get().getFlowEncapsulationType());
         }
+        if (requestedFlow.getPathComputationStrategy() == null) {
+            requestedFlow.setPathComputationStrategy(kildaConfigurationRepository.get().getPathComputationStrategy());
```

Comment:
Please, don't forget to make the same in FlowUpdateService.

Reply:
Rebased and added.